visualisation/aequatus/templates/aequatus.mako

```diff
@@ -0,0 +1,233 @@
+<%
+    default_title = "Aequatus of"
+
+    # Use root for resource loading.
+    root = h.url_for( '/' )
+    history_id = trans.security.encode_id( trans.history.id )
+    app_root    = root + "plugins/visualizations/aequatus/static/"
+%>
+## ----------------------------------------------------------------------------
+
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<title> ${visualization_name}</title>
+
+## install shared libraries
+        ${h.js( 'libs/jquery/jquery',
+                'libs/jquery/jquery.migrate',
+                'libs/jquery/jquery-ui',
+                'libs/bootstrap',
+                'libs/d3')}
+
+## aequatus-vis
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/init.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/geneView.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/drawGene.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/drawGeneExonOnly.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/drawCIGARs.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/util.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/d3_tree.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/newick.js" )}
+
+
+## aequatus plugin script
+        ${h.javascript_link( app_root + "controls.js" )}
+        ${h.javascript_link( app_root + "popup.js" )}
+
+
+## external scripts
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/scriptaculous/prototype.js" )}
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/jquery/js/jquery.svg.js" )}
+
+
+## external css
+        ${h.stylesheet_link( app_root + "aequatus-vis/scripts/jquery/jquery.svg.css" )}
+        ${h.stylesheet_link( app_root + "aequatus-vis/styles/font-awesome-4.2.0/css/font-awesome.css" )}
+        ${h.stylesheet_link( app_root + "aequatus-vis/styles/style.css" )}
+
+## aequatus css
+        ${h.stylesheet_link( app_root + "aequatus.css" )}
+       
+</head>
+
+## ----------------------------------------------------------------------------
+<body style="cursor: auto; height: 100%; position: absolute; width: 100%; z-index: 1999;">
+
+<script>
+
+        console.log("here")
+
+        kickOff();
+
+        var history_id = '${ trans.security.encode_id( trans.history.id ) }'
+
+        var hda_id = '${ trans.security.encode_id( hda.id ) }'
+
+        var ajax_url = '/api/histories/' + '${ trans.security.encode_id( trans.history.id ) }/contents/' + '${ trans.security.encode_id( hda.id ) }'+ '/display'
```

Comment:
I think this may be broken when the Galaxy URL has a prefix (e.g. `http://example.com/galaxy/`). Does the following work for you?
```javascript
var ajax_url = "${h.url_for( controller='/datasets', action='index')}/" + hda_id + "/display"
```